ocaml/tests/test_cluster.ml

```diff
@@ -37,7 +37,11 @@ let test_rpc ~__context call =
   | name, params ->
      failwith (Printf.sprintf "Unexpected RPC: %s(%s)" name (String.concat " " (List.map Rpc.to_string params)))
 
-let create_cluster ~__context =
+let create_cluster ~__context ?cluster_stack () =
+  let cluster_stack = match cluster_stack with
```

Comment:
You can use ?(cluster_stack=Constants.default_smapiv3_cluster_stack) instead of this match